extra/nrnivmodl_core_makefile.in

```diff
@@ -38,11 +38,6 @@ MOD_OBJS_DIR = $(OUTPUT_DIR)/corenrn/build
 
 # Linked libraries gathered by CMake
 LDFLAGS = $(LINKFLAGS) @CORENRN_COMMON_LDFLAGS@
-CORENRNLIB_FLAGS = -L$(CORENRN_LIB_DIR) -lcoreneuron
-CORENRNLIB_FLAGS += $(if @reportinglib_LIB_DIR@, -W$(subst ;, -W,l,-rpath,@reportinglib_LIB_DIR@),)
-CORENRNLIB_FLAGS += $(if @sonatareport_LIB_DIR@, -W$(subst ;, -W,l,-rpath,@sonatareport_LIB_DIR@),)
-CORENRNLIB_FLAGS += $(if @caliper_LIB_DIR@, -W$(subst ;, -W,l,-rpath,@caliper_LIB_DIR@),)
-CORENRNLIB_FLAGS += $(if @caliper_LIB_DIR@,-L@caliper_LIB_DIR@,)
```

Comment:
to clarify - has anything specifically changed in this PR that would make this unnecessary? I think Spack based build will add rpaths (?). IIRC, the reason that Jorge added this was that if you build neuron+coreneuron manually with reportinglib/sonata then special won't have rpaths embedded, and then without setting (DL)LD_LIBRRARY_PATH, special won't run.

Reply:
In principle the rpath arguments are generated automatically now: https://github.com/BlueBrain/CoreNeuron/blob/aa893f6c02f61f3efdc45d28146e63c7eeed9b10/CMake/MakefileBuildOptions.cmake#L63

Reply:
oh ok! 👌